polygeist/tools/cgeist/Test/Verification/sycl/builtins.cpp

```diff
@@ -0,0 +1,165 @@
+// RUN: clang++ -fsycl -fsycl-device-only -fsycl-targets=spir64-unknown-unknown-syclmlir -O0 -w -emit-mlir -o - %s | FileCheck %s
+// RUN: clang++ -fsycl -fsycl-device-only -fsycl-targets=spir64-unknown-unknown-syclmlir -O0 -w -emit-llvm -S -o %t %s && rm %t
```

Comment:
Do we need this RUN line ?

Reply:
I think it is to ensure there is no build failures.

Reply:
Yes, just to check we can lower to LLVM.

Reply:
OK there is some value in that. I was expecting to see checks for LLVM IR.